python/pyspark/sql/types.py

```diff
@@ -1694,6 +1694,21 @@ def from_arrow_schema(arrow_schema):
          for field in arrow_schema])
 
 
+def _correct_date_of_dataframe_from_arrow(pdf, schema):
```

Comment:
to be consistent with other methods in this file, how about `_check_dataframe_convert_date`

Reply:
Sure. I'll update it.